main.tf

```diff
@@ -1,9 +1,12 @@
 resource "aws_cloudtrail" "additional_auditing_trail" {
-  count                 = var.additional_auditing_trail != null ? 1 : 0
-  name                  = var.additional_auditing_trail.name
-  s3_bucket_name        = var.additional_auditing_trail.bucket
-  is_organization_trail = true
-  tags                  = var.tags
+  count                      = var.additional_auditing_trail != null ? 1 : 0
+  name                       = var.additional_auditing_trail.name
+  enable_log_file_validation = true
+  is_multi_region_trail      = true
+  is_organization_trail      = true
+  kms_key_id                 = module.kms_key.id
```

Comment:
I believe we may have issues by encrypting the logs since the buckets may be in a different account that doesn’t have access to the key

Reply:
Move these changes to new PR: https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/pull/101